modules/nf-core/methyldackel/extract/main.nf

```diff
@@ -13,7 +13,8 @@ process METHYLDACKEL_EXTRACT {
     path fai
 
     output:
-    tuple val(meta), path("*.bedGraph"), emit: bedgraph
+    tuple val(meta), path("*.bedGraph"),  optional: true, emit: bedgraph
+    tuple val(meta), path("*.methylKit"), optional: true, emit: methylkit
     path  "versions.yml"               , emit: versions
```

Comment:
![image](https://user-images.githubusercontent.com/20095261/199849445-7f2ae0f0-94c1-42a7-b534-2881b210286c.png)
![image](https://user-images.githubusercontent.com/20095261/199849466-9708ce40-a0c4-4827-bd42-5612b6ff40b3.png)
Idk which one's actually out of line. :upside_down_face: You'll figure it out!
```suggestion
    tuple val(meta), path("*.bedGraph"),  optional: true, emit: bedgraph
    tuple val(meta), path("*.methylKit") , optional: true, emit: methylkit
    path  "versions.yml"                        , emit: versions
```

Reply:
You mean to align the `emit`? Like this?

```suggestion
    tuple val(meta), path("*.bedGraph"),  optional: true, emit: bedgraph
    tuple val(meta), path("*.methylKit"), optional: true, emit: methylkit
    path  "versions.yml"                                , emit: versions
```